### PR TITLE
Renaming vars and things

### PR DESCRIPTION
**Pull request by misterbisson**
- Changing all the varnames to triton
  - but, ironically, using `SDC_*` as the env var prefix
- "Path, rather than "File"
  - For consistency with langauge used in https://www.terraform.io/docs/providers/triton/index.html

### CODE REVIEW

File: driver.go [hunk 1]
```diff
@@ -23,66 +23,67 @@ import (
 const (
 	driverName = "triton"
 	flagPrefix = driverName + "-"
-	envPrefix  = "TRITON_"
+    // SDC_ is for historical reasons
```
Comment (tianon) [inline driver.go]: This should be tabbed, not spaces. :+1:

**Comment by tianon** [comment]
I was just about to make a very similar commit to this PR (although not the `TRITON_` back to `SDC_` -- doesn't the `triton` tool support both now?) :smile:

There's also an embarrassing compilation error I introduced in https://github.com/joyent/docker-machine-driver-triton/commit/340765533e4888fed7e6d0c17f09d3fee2a9e565. :sheep:

**Comment by misterbisson** [comment]
> doesn't the `triton` tool support both now?

It may read both, but `triton env` only sets `SDC_*`. That could be fixed, I guess.

**Comment by tianon** [comment]
Ah, I see.  Legacy is fun. :disappointed:

LGTM

File: driver.go [hunk 2]
```diff
@@ -96,58 +97,58 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
...
 			EnvVar: envPrefix + "ACCOUNT",
 			Name:   flagPrefix + "account",
 			Usage:  "Login name/username",
-			Value:  defaultSdcAccount,
+			Value:  defaultTritonAccount,
 		},
 		mcnflag.StringFlag{
 			EnvVar: envPrefix + "KEY_ID",
 			Name:   flagPrefix + "key-id",
 			Usage:  fmt.Sprintf(`The fingerprint of $%sKEY_FILE (ssh-keygen -l -E md5 -f $%sKEY_FILE | awk '{ gsub(/^[^:]+:/, "", $2); print $2 }')`, envPrefix, envPrefix),
```
Comment (tianon) [inline driver.go]: Oh, this has a few more cases of `_FILE` that should be `_PATH`

File: driver.go [hunk 3]
```diff
@@ -96,58 +97,58 @@ func (d *Driver) GetCreateFlags() []mcnflag.Flag {
...
-			Value:  defaultSdcKeyId,
+			Value:  defaultTritonKeyId,
 		},
 		mcnflag.StringFlag{
 			EnvVar: envPrefix + "KEY_FILE",
 			Name:   flagPrefix + "key-file",
```
Comment (tianon) [inline driver.go]: And these probably should be too :smile:

File: driver.go [hunk 4]
```diff
@@ -23,66 +23,69 @@ import (
...
+	defaultTritonKeyId   = ""
+	defaultTritonUrl     = "https://us-east-1.api.joyent.com"
 
 	// https://docs.joyent.com/public-cloud/instances/virtual-machines/images/linux/debian#debian-8
-	defaultSdcImage   = "debian-8"
-	defaultSdcPackage = "g3-standard-0.25-kvm"
+	defaultTritonImage   = "debian-8"
+	defaultTritonPackage = "g3-standard-0.25-kvm"
+
+	errUnimplemented = errors.New("UNIMPLEMENTED")
```
Comment (misterbisson) [inline driver.go]: @tianon you were suggesting I add this back in https://github.com/joyent/docker-machine-driver-triton/pull/1#issuecomment-213502583 ?

Comment (tianon) [inline driver.go]: No, just noting that fixing the compilation error I'd introduced was something else I'd staged locally (already pushed the fix in https://github.com/joyent/docker-machine-driver-triton/commit/5456b1f15a4e26d195dad0b5df2019569a40457d).  Sorry for the confusion!

**Comment by tianon** [comment]
LGTM :+1:

**Pull request merged**